src/quick-start.md

```diff
@@ -28,13 +28,32 @@ helm repo add kubewarden https://charts.kubewarden.io
 helm install --wait -n kubewarden --create-namespace kubewarden-crds kubewarden/kubewarden-crds
 
 helm install --wait -n kubewarden kubewarden-controller kubewarden/kubewarden-controller
+
+helm install --wait -n kubewarden kubewarden-controller kubewarden/kubewarden-defaults
 ```
 
 The following charts should be installed inside the `kubewarden` namespace in your Kubernetes cluster:
 
 * `kubewarden-crds`, which will register the `ClusterAdmissionPolicy` and `PolicyServer` Custom Resource Definitions
 
-* `kubewarden-controller` with a default configuration, and which will create a `PolicyServer` resource named `default`. 
+* `kubewarden-controller`, it installs the Kubewarden controller
+
+* `kubewarden-defaults`, it creates a `PolicyServer` resource named `default`.
+
+
+> **WARNING:**
+> Since `kubewarden-controller` version `v0.4.0` no Policy Server is installed in the chart.
```

Comment:
```suggestion
> Since [`v0.4.0`](https://github.com/kubewarden/kubewarden-controller/releases/tag/v0.4.0), a PolicyServer resource named default will not be created using the `kubewarden-controller` chart.
```